src/chart.zoom.js

```diff
@@ -186,30 +198,97 @@ zoomNS.panFunctions.logarithmic = panNumericalScale;
 
 // Chartjs Zoom Plugin
 var zoomPlugin = {
+	afterInit: function(chartInstance) {
+		chartInstance.resetZoom = function() {
```

Comment:
It'd be nice if the `resetZoom` function could store any settings that the user had before hand. Then if the user had options at chart creation, this function could put them back rather than delete them.